File: next/security/user and permission/backend-ms1/src/main/java/ma/zs/stocky/ws/dto/security/RoleDto.java
package  ma.zs.stocky.ws.dto.security;

import ma.zs.stocky.zynerator.audit.Log;
import ma.zs.stocky.zynerator.dto.AuditBaseDto;
import com.fasterxml.jackson.annotation.JsonInclude;





@JsonInclude(JsonInclude.Include.NON_NULL)
public class RoleDto  extends AuditBaseDto {

    private String authority  ;




    public RoleDto(){
        super();
    }



    @Log
    public String getAuthority(){
        return this.authority;
    }
    public void setAuthority(String authority){
        this.authority = authority;
    }








}

File: laravel/v3/backend-ms1/app/Dto/security/RoleDto.php
<?php

namespace App\Dto\security;


class RoleDto
{

    private $authority;




    public function getId() {
        return $this->id;
    }
    public function setId($id){
        $this->id = $id;
    }

    public function getAuthority() {
        return $this->authority;
    }
    public function setAuthority($authority){
        $this->authority = $authority;
    }

}


